qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/create/offer/SelectItemsView.groovy

```diff
@@ -230,11 +234,21 @@ class SelectItemsView extends VerticalLayout{
      * This method adds the retrieved Customer Information to the Customer grid
      */
     private void setupDataProvider() {
-        this.sequencingGrid.setItems(sequencingProduct)
-        this.projectManagementGrid.setItems(projectManagementProduct)
-        this.primaryAnalyseGrid.setItems(primaryAnalyseProduct)
-        this.secondaryAnalyseGrid.setItems(secondaryAnalyseProduct)
-        this.storageGrid.setItems(storageProduct)
+        ListDataProvider<ProductItemViewModel> sequencingProductDataProvider = new ListDataProvider(createOfferViewModel.sequencingProducts)
+        this.sequencingGrid.setDataProvider(sequencingProductDataProvider)
+
+        ListDataProvider<ProductItemViewModel> managementProductDataProvider = new ListDataProvider(createOfferViewModel.managementProducts)
+        this.projectManagementGrid.setDataProvider(managementProductDataProvider)
+
+        ListDataProvider<ProductItemViewModel> primaryAnalysisProductDataProvider = new ListDataProvider(createOfferViewModel.primaryAnalysisProducts)
+        this.primaryAnalyseGrid.setDataProvider(primaryAnalysisProductDataProvider)
+
+        ListDataProvider<ProductItemViewModel> secondaryAnalysisProductDataProvider = new ListDataProvider(createOfferViewModel.secondaryAnalysisProducts)
+        this.secondaryAnalyseGrid.setDataProvider(secondaryAnalysisProductDataProvider)
+
+        ListDataProvider<ProductItemViewModel> storageProductDataProvider = new ListDataProvider(createOfferViewModel.storageProducts)
+        this.storageGrid.setDataProvider(storageProductDataProvider)
+
         this.overviewGrid.setItems(selectedItems)
```

Comment:
```suggestion
ListDataProvider<ProductItemViewModel> selectedItemsDataProvider = new ListDataProvider(createOfferViewModel.selectedItems)
        this.overviewGrid.setDataProvider(selectedItemsDataProvider)
```
Also do the same for the selected items. This should also be in the view model I think.